Packs/MicrosoftGraphGroups/Integrations/MicrosoftGraphGroups/MicrosoftGraphGroups.py

```diff
@@ -311,7 +139,7 @@ def delete_group(self, group_id: str):
         """
         #  If successful, this method returns 204 No Content response code.
         #  It does not return anything in the response body.
-        self.http_request('DELETE ', f'groups/{group_id}')
+        self.ms_client.http_request(method='DELETE ', url_suffix=f'groups/{group_id}', resp_type="text")
```

Comment:
why `text` ?

Reply:
to avoid parsing errors. there is no content in those kinds of requests. Is it better to use `resp_type='response'` instead?

Reply:
its ok, lets just add it to the comment above in the code